### PR TITLE
feat: add header

### PR DESCRIPTION
**Pull request by DubichTonya**
1. Add Header
2. Add eslint-config-prettier

### CODE REVIEW

File: src/assets/images/icons/header/game.svg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (hhharm) [inline src/assets/images/icons/header/game.svg:1]: Please, use some minification tool, for this and other images. You can find such tool in google or here https://css-tricks.com/tools-for-optimizing-svg/

File: src/app/core/components/header/header.component.ts [hunk 2]
```diff
@@ -1,10 +1,51 @@
 import { Component } from '@angular/core';
 
+export interface IHeaderMenu {
```
Comment (hhharm) [inline src/app/core/components/header/header.component.ts:3]: it should be in models/types/etc folder (i'm not sure how you named it)

File: src/app/core/components/header/header.component.ts [hunk 3]
```diff
@@ -1,10 +1,51 @@
...
-    constructor() {}
+  iconImagePath = '../../../../assets/images/icons/header/';
+  isAuthorization = true;
+  userPhoto = 'https://ofigenno.com/wp-content/uploads/posts/z/zoi-saldana/1.jpg';
+  iconHome = `${this.iconImagePath}home.svg`;
+  iconTextbook = `${this.iconImagePath}textbook.svg`;
+  iconMiniGames = `${this.iconImagePath}game.svg`;
+  iconStatistic = `${this.iconImagePath}statistics.svg`;
+
+  headerMenu = [
```
Comment (hhharm) [inline src/app/core/components/header/header.component.ts:23]: move it to const, please

File: src/app/core/components/header/header.component.ts [hunk 4]
```diff
@@ -1,10 +1,51 @@
...
-    styleUrls: ['./header.component.scss'],
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
 })
 export class HeaderComponent {
-    constructor() {}
+  iconImagePath = '../../../../assets/images/icons/header/';
+  isAuthorization = true;
+  userPhoto = 'https://ofigenno.com/wp-content/uploads/posts/z/zoi-saldana/1.jpg';
```
Comment (hhharm) [inline src/app/core/components/header/header.component.ts:17]: Don't use links to external media, it's not robust. If you need it, save to assets and take it from there

File: src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss [hunk 5]
```diff
@@ -1,7 +1,6 @@
 :host {
   position: relative;
-  background: #344557 url(https://unsplash.imgix.net/photo-1423683249427-8ca22bd873e0?fit=crop&fm=jpg&h=700&q=75&w=1050)
-    0 0 no-repeat;
+  background: #344557 url("https://unsplash.imgix.net/photo-1423683249427-8ca22bd873e0?fit=crop&fm=jpg&h=700&q=75&w=1050") 0 0 no-repeat;
```
Comment (hhharm) [inline src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss:3]: same about external link

**Review by rss-maksim — APPROVED** [review]

File: src/assets/images/icons/header/game.svg (hunk 1 above)
Comment (DubichTonya) [reply src/assets/images/icons/header/game.svg]: Done

File: src/app/core/components/header/header.component.ts (hunk 2 above)
Comment (DubichTonya) [reply src/app/core/components/header/header.component.ts]: Done

File: src/app/core/components/header/header.component.ts (hunk 3 above)
Comment (DubichTonya) [reply src/app/core/components/header/header.component.ts:15]: Done

File: src/app/core/components/header/header.component.ts (hunk 4 above)
Comment (DubichTonya) [reply src/app/core/components/header/header.component.ts]: Done

File: src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss (hunk 5 above)
Comment (DubichTonya) [reply src/app/mini-games/pages/mini-games-page/mini-games-page.component.scss]: Done

**Review by Ars6300 — APPROVED** [review]

**Pull request merged**